database/index.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require ('mongoose');
+mongoose.connect('mongodb://localhost/houses', {} , (err)=>{
+  if (err) {
+    console.log(err);
+  }
+  console.log('im connected?');
+});
+
+let repoSchema = mongoose.Schema({
+  _id: {type: Number, unique: true, index: true},
+  postTitle: Text,
+  cost: Number,
+  image: Text,
+  roomtype: {type: Text, beds: Number, Premium: Boolean},
+  rating: {StarNum: Number, Votes: Numbers},
+  description: Text
+});
+
+let repo = mongoose.model('Similar', repoSchema);
```

Comment:
I would recommend calling repo something more specific. Maybe something like Carousel. Model is also a class so by convention, we should capitalize the first letter.

'Similar' will become your collection name so name it something meaningful. 

```suggestion
 let Carousel = mongoose.model('Similar', repoSchema);
```